master/reference/advanced/etcd-rbac/calico-etcdv2-paths.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Calico key path prefixes in etcd v2
+---
+
+The Paths listed here are the path prefixes that a particular calico component
+needs access to in etcd to function successfully.
+
+## calico/node
+
+XXX Does this change if it is only being used for with docker (libnetwork)?
+
+XXX Does this change any if calico/node is only being used for kubernetes?
```

Comment:
I think only if you aren't using Calico IPAM, but I think you generally would if you are using the etcd datastore driver.